<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wind Waker Randomizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>870</width>
         <height>658</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_randomizer_settings">
           <attribute name="title">
            <string>Randomizer Settings</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="1" column="2">
               <widget class="QPushButton" name="output_folder_browse_button">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="clean_iso_path"/>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="output_folder"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_for_output_folder">
                <property name="text">
                 <string>Randomized Output Folder</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="clean_iso_path_browse_button">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_for_clean_iso_path">
                <property name="text">
                 <string>Vanilla Wind Waker ISO [[?]](help)</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::MarkdownText</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="plando_file_browse_button">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="plando_file"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_for_plando_file">
                <property name="text">
                 <string>APTWW File [[?]](help)</string>
                </property>
                <property name="textFormat">
                 <enum>Qt::MarkdownText</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Convenience Tweaks</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QCheckBox" name="remove_title_and_ending_videos">
                 <property name="text">
                  <string>Remove Title and Ending Videos</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="invert_sea_compass_x_axis">
                 <property name="text">
                  <string>Invert Sea Compass X-Axis</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QCheckBox" name="invert_camera_x_axis">
                 <property name="text">
                  <string>Invert Camera X-Axis</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_7">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="CosmeticTab" name="tab_player_customization">
           <attribute name="title">
            <string>Player Customization</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="option_description">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="randomize_button">
      <property name="text">
       <string>Randomize</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CosmeticTab</class>
   <extends>QWidget</extends>
   <header>wwr_ui.cosmetic_tab</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>clean_iso_path</tabstop>
  <tabstop>clean_iso_path_browse_button</tabstop>
  <tabstop>output_folder</tabstop>
  <tabstop>output_folder_browse_button</tabstop>
  <tabstop>plando_file</tabstop>
  <tabstop>plando_file_browse_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
